File: CIVITAS/src/civitas/OperacionInmobiliaria.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package civitas;

/**
 *
 * @author Usuario
 */
public class OperacionInmobiliaria {
    private int numPropiedad;
    private GestionesInmobiliarias gestion;

    public OperacionInmobiliaria( GestionesInmobiliarias gestion , int numPropiedad) {
        this.numPropiedad = numPropiedad;
        this.gestion = gestion;
    }

    public int getNumPropiedad() {
        return numPropiedad;
    }

    public GestionesInmobiliarias getGestion() {
        return gestion;
    }
    
    
}

File: CIVITASRUBY/lib/operacion_inmobiliaria.rb
# To change this license header, choose License Headers in Project Properties.
# To change this template file, choose Tools | Templates
# and open the template in the editor.


module Civitas
  class Operacion_Inmobiliaria
    attr_reader :gestion, :num_propiedad
    def initialize(gestion,num_propiedad)
      @num_propiedad = num_propiedad
      @gestion = gestion
    end
    
  end
end